src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Public.Data/PreviewTokenService.cs

```diff
@@ -38,7 +40,9 @@ public async Task<Either<ActionResult, PreviewTokenViewModel>> CreatePreviewToke
                 {
                     DataSetVersionId = dataSetVersionId,
                     Label = label,
-                    Expiry = DateTimeOffset.UtcNow.AddDays(1),
+                    Created = DateTimeOffset.UtcNow,
+                    Activates = activates,
```

Comment:
See my other comment about possibly making `Activates` non-nullable, in which case it would be something like 

```csharp
activates ?? DateTimeOffset.UtcNow`
```

to make it the same as `Created`.